### ISSUE TITLE
[Bug] fzf exe not available in scoop\apps\fzf\current after update to 0.48.1

### ISSUE THREAD

**Issue by aniketgm** [ScoopInstaller/Main#5640]
#### Current Behavior
fzf.exe is not available after update to 0.48.1

#### Expected Behavior
fzf.exe should be available to use after update.

#### Additional context/output
![image](https://github.com/ScoopInstaller/Scoop/assets/14215698/7952da7c-fe27-4835-94ad-8cda2cab3c2a)

### System details

**Windows version:** Not required

**OS architecture:** Not required

**PowerShell version:** Not required

**Additional software:** Not required

#### Scoop Configuration
Not required

**Comment by aliesbelik** [comment]
May relate to junegunn/fzf#3690.

**Comment by aliesbelik** [comment]
> I believe the person who updated scoop repo with latest fzf 0.48.1 missed to update the exe.

As you could [easily see](https://github.com/ScoopInstaller/Main/commits/master/bucket/fzf.json) all the updates for the last versions were made automatically by bot, there are no changes neither in release artifact, nor in binary names.

> I'm not sure how that related to this bug issue. Here the exe itself is not available.

Depending on the antivirus software you are using and its settings it could block installation or quarantine binary in case of threat detection (see e.g. ScoopInstaller/Main#4430). Ensure first that it's not your case.

**Comment by aniketgm** [comment]
> Depending on the antivirus software you are using and its settings it could block installation or quarantine binary in case of threat detection (see e.g. https://github.com/ScoopInstaller/Main/issues/4430). Ensure first that it's not your case.

Ah, now I see. Thank you the explanation. I didn't know anti-virus silently did such things. Now I'm suspicious of the AV as well.
I did not get any `Wacatac` virus or any other warning/notification and the exe seems to be blocked when updating. (similar to #4430).

I tried by disabling the anti-virus and re-updating `fzf` to `0.48.1`. It installed the exe, but after reenabling the AV, the AV started to give me warnings. So I'm reverting back to `0.46.1` for now, which was my previous installed version.

For others, recommendation is same. Revert back to whichever previous fzf version you have for now, until https://github.com/junegunn/fzf/issues/3690 has some solution/workaround. ( On how to revert back to previous version see https://github.com/ScoopInstaller/Main/issues/5640#issuecomment-2015468116 )
